libs/community/langchain_community/embeddings/fastembed.py

```diff
@@ -12,6 +12,7 @@
 
 class FastEmbedEmbeddings(BaseModel, Embeddings):
```

Comment:
let's deprecate this

Reply:
@baskaryan ?

Reply:
how was this resolved?

Reply:
Yes, I removed FastEmbedEmbeddings class reference from qdrant.mdx page.
If you mean the deprecate decorator for this class, I assume this change should be implemented in another PR (this PR is only about doc changes).